eq-author-api/schema/tests/answerGroupValidation.test.js

```diff
@@ -81,23 +80,23 @@ describe("Answer group validation", () => {
           },
         ],
       });
-
+      questionnaire = ctx.questionnaire;
       const pageId = questionnaire.sections[0].pages[0].id;
-      const page = await queryPage(questionnaire, pageId);
+      const page = await queryPage(ctx, pageId);
 
       expect(page.totalValidation).not.toBeNull();
 
-      await createAnswer(questionnaire, {
+      await createAnswer(ctx, {
         questionPageId: pageId,
         type: CURRENCY,
       });
 
-      const updatedPage = await queryPage(questionnaire, pageId);
+      const updatedPage = await queryPage(ctx, pageId);
       expect(updatedPage.totalValidation).toBeNull();
     });
 
     it("should not add it even when a second is added", async () => {
```

Comment:
could this line be more explanatory?

Reply:
In what way? This line change is just due to requiring the whole ctx object to run queries and mutations?

Reply:
I meant the test description.